src/init.cpp

```diff
@@ -474,12 +474,12 @@ std::string HelpMessage(HelpMessageMode mode)
         CURRENCY_UNIT, FormatMoney(DEFAULT_MIN_RELAY_TX_FEE)));
     strUsage += HelpMessageOpt("-maxtxfee=<amt>", strprintf(_("Maximum total fees (in %s) to use in a single wallet transaction or raw transaction; setting this too low may abort large transactions (default: %s)"),
         CURRENCY_UNIT, FormatMoney(DEFAULT_TRANSACTION_MAXFEE)));
-    strUsage += HelpMessageOpt("-printtoconsole", strprintf(_("Send trace/debug info to console instead of %s"), GetDebugLogPath()));
+    strUsage += HelpMessageOpt("-printtoconsole", _("Send trace/debug info to console instead of debug log"));
```

Comment:
```suggestion
    strUsage += HelpMessageOpt("-printtoconsole", _("Send trace/debug info to console instead of the configured log file."));
```

Reply:
Ah, that’s better. But none of the other cases have a final period, so I’ll commit without that.